graphql_compiler/query_formatting/common.py

```diff
@@ -28,6 +28,63 @@ def _raise_invalid_type_error(name, expected_python_type_name, value):
                                                          type(value).__name__))
 
 
+def _deserialize_anonymous_json_argument(expected_type, value):
+    """Deserialize argument. See docstring of deserialize_json_argument."""
```

Comment:
Can you add `Args/Returns/Raises` sections? Mostly interested in the `Raises` here so we know what errors to catch and rewrap in the caller function.